src/11_distance_nearest_port_dev.jl

```diff
@@ -0,0 +1,29 @@
+using Distances
+
+include("common.jl")
+
+Ports = GDF.read("c:/Users/bgrier/Documents/QLD_ports_mercator_via_MP/ports_QLD_merc.shp")
```

Comment:
Please do not use absolute paths in these scripts. No one else has this directory.